src/js/templates.js

```diff
@@ -972,6 +972,47 @@ let playerView = new Vue({
         openInvidious: (videoId) => {
             shell.openExternal(invidiousInstance + '/watch?v=' + videoId);
         },
+        openInVLC: (videoId) => {
+            // getting YouTube url as usual
+            const url = 'https://youtube.com/watch?v=' + videoId;
+
+            // before VLC started we will set html player to pause
+            const htmlPlayer = $('.videoPlayer').get(0);
```

Comment:
This function crashes the app when you try to view a video with the Autoplay setting enabled.  You'll need to update this to check for the `player` variable as well.  This logic can be found elsewhere in the app and should fix this issue.

```javascript
    if (typeof (player) !== 'undefined') {
        videoPlayer = player;
    } else {
        videoPlayer = $('.videoPlayer').get(0);
    }
```